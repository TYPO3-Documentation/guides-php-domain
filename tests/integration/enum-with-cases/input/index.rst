===================
PHP Enum with Const
===================

.. php:enum:: Suit

    In playing cards, a suit is one of the categories into which the
    cards of a deck are divided.

    .. php:case:: Hearts

        Hearts is one of the four suits in playing cards.

    .. php:case:: Diamonds

        Diamonds is one of the four suits in playing cards.

    .. php:case:: Clubs

        Clubs is one of the four suits in playing cards.

    .. php:case:: Spades

        Spades is one of the four suits in playing cards.
