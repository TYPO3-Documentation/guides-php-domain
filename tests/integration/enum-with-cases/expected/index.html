<!-- content start -->
    <div class="section" id="php-enum-with-const">
    <h1>PHP Enum with Const</h1>

            <dl class="php enum">
    <dt class="sig sig-object php"
        id="suit">
        <em class="property"><span class="pre">enum</span> </em>
        <span class="sig-name descname"><span class="pre">Suit</span></span>

    </dt>
    <dd>
        <p>In playing cards, a suit is one of the categories into which the
cards of a deck are divided.</p><dl class="php case">
    <dt class="sig sig-object php" id="hearts">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Hearts</span></span>
 : </dt>
    <dd><p>Hearts is one of the four suits in playing cards.</p></dd>
</dl>
<dl class="php case">
    <dt class="sig sig-object php" id="diamonds">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Diamonds</span></span>
 : </dt>
    <dd><p>Diamonds is one of the four suits in playing cards.</p></dd>
</dl>
<dl class="php case">
    <dt class="sig sig-object php" id="clubs">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Clubs</span></span>
 : </dt>
    <dd><p>Clubs is one of the four suits in playing cards.</p></dd>
</dl>
<dl class="php case">
    <dt class="sig sig-object php" id="spades">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Spades</span></span>
 : </dt>
    <dd><p>Spades is one of the four suits in playing cards.</p></dd>
</dl>

    </dd>
</dl>

    </div>

<!-- content end -->
